src/shell/services/instance.ts

```diff
@@ -695,6 +702,62 @@ export const instanceApi = createApi({
       }),
       invalidatesTags: ["Groups"],
     }),
+    //Get instance workflow status labels (active/deactivated)
```

Comment:
I believe I already created a getter for this as well called `getWorkflowStatusLabels`, can you just use that instead and just update it to support the showDeleted query param?